### DISCUSSION TITLE
FastPitch VCTK pre-trained model

### DISCUSSION THREAD

**Discussion by adrianastan** [coqui-ai/TTS#1312] [General Q&A]
Hi, 

I found on your release page: https://github.com/coqui-ai/TTS/releases that in v.0.4.0 there should be a FastPitch trained on the VCTK data. Is this model available somewhere?

Thanks!

**Comment by erogol** [comment]
If you run

```
tts --list_models
```

you see it as the 12th model.

**Comment by adrianastan** [comment]
Oh, sorry about that! I managed to use it within coqui.

I am now wondering if there is any vocoder_model which works better with the FastPitch model. I tried: 
`vocoder_models/universal/libri-tts/wavegrad`
`vocoder_models/en/vctk/hifigan_v2`
`vocoder_models/universal/libri-tts/fullband-melgan`

But none of them seem to be any better than the Griffin-Lim. Actually the hifigan_v2 generates just noise which I assume is  problem of compatibility with the mel spectrograms.

Thanks!

**Reply by erogol** [reply]
There is no compatible vocoder yet for this model.

**Reply by adrianastan** [reply]
Ok, thank you!

**Reply by yurtmete** [reply]
Hi! 

First of all, great resource! I admire the work and suggesting it to everybody :) 

Second, I was also wondering on this. Is there any update now? I assume still the best option for FastPitch model is Griffin-Lim in Coqui universe? 

Thank you in advance.
